composer.json

```diff
@@ -24,7 +24,7 @@
     "require-dev": {
         "symfony-cmf/routing-bundle": "~1.2",
         "symfony-cmf/testing": "~1.1",
-        "sonata-project/doctrine-phpcr-admin-bundle": "~1.1",
+        "sonata-project/doctrine-phpcr-admin-bundle": "~2.0",
```

Comment:
we need 2.0, as we need SonataAdminBundle 2.4 (we can use 2.3, but it'll look not so nice and extensions won't work over there).